Gemfile

```diff
@@ -1,6 +1,4 @@
-require File.expand_path('Gemfile.global.rb', File.dirname(__FILE__))
-
-MiqBundler.include_gemfile("gems/pending/Gemfile", binding)
+eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
```

Comment:
Strange, is it ours?  I don't remember it or why it wasn't being used in the first place.  Nit: and the commit message didn't help me either.  Can you clarify that in the commit message?  Thanks!